### ISSUE TITLE
How to use detectron2 maskrcnn trained weights with torchvision maskrcnn

### ISSUE THREAD

**Issue by MrGrayCode** [facebookresearch/detectron2#777]
I have maskrcnn model weigths trained using detectron2. I wish to use the same weights with the [torchvision maskrcnn](https://pytorch.org/docs/stable/torchvision/models.html#mask-r-cnn) model. Loading the model directly gives keyerror.

**Comment by ppwwyyxx** [comment]
You cannot, as the model definition is not compatible (both in the math and in the names).
